### ISSUE TITLE
Display of types for variants in overview pages

### ISSUE THREAD

**Issue by mbrush** [monarch-initiative/monarch-legacy#921]
The 'types' displayed for  variants in their Overview page are not informative or understandable for most users.  For example, for DARS, MET256LEU  (http://beta.monarchinitiative.org/variant/ClinVarVariant:50988) we see the types as a list:  "variant locus, sequence alteration, sequence feature".  For now, it might be best to only display the leaf node types (and not inferred types along path to root as seems to be the case now)

Ultimately I would like to see something akin to a hierarchical display of these types in their respective ontology with the path(s) to root from most granular class bolded. And these bolded classes representing variant types should be links that resolve to a definition. 

Basically, this would make the typing of entities that are instances in our data (such as variants) render in a similar way as the classification of entities that are classes (such as phenotypes or  diseases) in their overview pages (e.g. http://beta.monarchinitiative.org/phenotype/HP:0003429)

**Comment by nlwashington** [comment]
this one ought to show "single nucleotide variant", which is linked via the :type or :subClassOf relationships.  can you add this, @kshefchek ?
